Packs/MicrosoftGraphMail/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### O365 Outlook Mail (Using Graph API)
+- Fixed an issue where the ***msgraph-mail-list-emails*** command failed to execute in case the *search* command arguments contained the double quotation character ( " ).
```

Comment:
```suggestion
- Fixed an issue where the ***msgraph-mail-list-emails*** command failed to execute in case the *search* command argument contained the double quotation character ( " ).
```